libraries/tock-tbf/src/types.rs

```diff
@@ -702,6 +704,66 @@ impl TbfHeader {
         }
     }
 
+    /// Get the process `write_id`.
+    /// Returns `None` if a `write_id` is not included.
+    pub fn get_write_id(&self) -> Option<u32> {
```

Comment:
Are these used?

Also, perhaps we could namespace these somehow? `get_write_id` is a little vague.